distributed/scheduler.py

```diff
@@ -8212,6 +8217,19 @@ def task(self) -> str:
     def last_worker(self) -> WorkerState:
         return self.args[1]
 
+    @property
+    def allowed_failures(self) -> int:
+        return self.args[2]
+
+    def __str__(self) -> str:
+        return (
+            f"Attempted to run task {self.task} on {self.allowed_failures} different "
+            "workers but was unsuccessful. The last worker that attempted to run the task was "
```

Comment:
```suggestion
            "workers, but all those workers died while running it. The last worker that attempted to run the task was "
```
"was unsuccessful" doesn't make it clear to me that the workers crashed (vs the task just failed)